io.github.koromelodev.mindmate.yml

```diff
@@ -0,0 +1,24 @@
+---
+app-id: io.github.koromelodev.mindmate
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: MindMate
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-config
```

Comment:
this is not allowed.

Reply:
How can I access the config folder?

Reply:
application have access to their config directory which is physically in `~/.var/app/$FLATPAK_ID` and proper XDG directories support will find it.

so what are you trying to do?